### DISCUSSION TITLE
Component token not getting applied

### DISCUSSION THREAD

**Discussion by susantpatel** [ant-design/ant-design#44730] [Q&A]
Component token siderBg for Layout is not getting applied

I want to change the background color of the sider and i am doing it like  this 

```
<ConfigProvider
    theme={{
      components: {
        Layout: {
          colorBgBase: "green",
          colorBgBody: "red",
          siderBg: "#00FF00",
        },
       
    }}
  >
```

TypeScript is throwing this error 

Type '{ colorBgBase: string; colorBgBody: string; siderBg: string; triggerBg: string; }' is not assignable to type 'Partial<ComponentToken> & Partial<AliasToken> & { algorithm?: boolean | MappingAlgorithm | MappingAlgorithm[] | undefined; }'.
  Object literal may only specify known properties, and 'siderBg' does not exist in type 'Partial<ComponentToken> & Partial<AliasToken> & { algorithm?: boolean | MappingAlgorithm | MappingAlgorithm[] | undefined; }'.ts(2322)

**Comment by MattinaYang** [comment]
same problem
